store/mongo/mongo_suite_test.go

```diff
@@ -0,0 +1,69 @@
+package mongo_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"fmt"
+	"testing"
+	"time"
+
+	mgo "gopkg.in/mgo.v2"
+
+	"github.com/tidepool-org/platform/app"
+)
+
+func TestSuite(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "store/mongo")
+}
+
+func MongoTestAddress() string {
+	return "127.0.0.1"
+}
+
+func MongoTestSession() *mgo.Session {
+	return _nodeSession
+}
+
+func MongoTestDatabase() string {
+	return _database
+}
+
+var (
+	_globalSession *mgo.Session
```

Comment:
This isn't a Go thing, but just something I started to do for private global variables to distinguish them from local function variables. Impossible to know whether `database` is a local variable or private global variable. If there isn't a way to distinguish the two then it is very easy to accidentally shadow a private global variable with a local variable.  For example:

```
var myVar string = "global"

func SetMyVar(newVar string) {
  myVar := newVar
}
```

Obviously a too-simple example (and it warn on not using the local variable `myVar`), but hopefully you get the idea.

In any case, just something I started to keep me straight on private global variables. Besides, there are *very* few, if any, of these in the code since our code has to be safe in a concurrent environment.